src/zenml/artifacts/unmaterialized_artifact.py

```diff
@@ -15,14 +15,14 @@
 
 
 from zenml.models import (
-    ArtifactResponse,
+    ArtifactVersionResponse,
     RunMetadataResponse,
     UserResponse,
     WorkspaceResponse,
 )
 
 
-class UnmaterializedArtifact(ArtifactResponse):
+class UnmaterializedArtifact(ArtifactVersionResponse):
```

Comment:
Now that we have `Artifact`s and `ArtifactVersion`s as separate concepts, the naming here might be slightly confusing. At the same time, `UnmaterializedArtifactVersion` also sounds a bit off.

Reply:
Yeah I didn't rename any user-facing concepts (`ExternalArtifact`, `UnmaterializedArtifact`, `ArtifactConfig`, `save_artifact`, `load_artifact`, `log_artifact_metadata`, ...) since versioning is not something we would want to expose to users at that level IMO